Unable to build simple rustls example (rayon-core can't find panic_unwind)

I am unable to build a simple `rustls` example using Xargo. Details below:

```
$ rustup show
...
active toolchain
----------------

nightly-x86_64-apple-darwin (default)
rustc 1.24.0-nightly (cfba0d446 2017-12-05)

$ xargo --version
xargo 0.3.8
cargo 0.24.0-nightly (5bb478a51 2017-11-29)
```

```toml
# Xargo.toml
[dependencies.std]
features = ["force_alloc_system"]
```

```toml
# Cargo.toml
[package]
name = "rustls-test"
version = "0.1.0"

[dependencies]
rustls = "0.11.0"

[profile.release]
panic = 'abort'
```

```rust
// main.rs
extern crate rustls;

fn main() {
    println!("Hello, World");
}
```

```
$ xargo build --release
...
error[E0463]: can't find crate for `panic_unwind`
error: aborting due to previous error
error: Could not compile `rayon-core`.
```

Is there a way to solve this issue?

--- a/README.md
+++ b/README.md
@@ -324,14 +324,14 @@ stage = 2
   without the `jemalloc` feature.  For some further information on this issue,
   see [rust-lang/rust#43637](https://github.com/rust-lang/rust/issues/43637#issuecomment-320463578).
 
-- When using compiler plugins (e.g. `serde_derive`) the target triple must be
-  provided even when compiling for the host platform due to the way cargo
-  handles compiler plugins. I.e., use `xargo build --target
-  x86_64-unknown-linux-gnu` instead of just `xargo
-  build`. (Surprisingly, these commands are NOT the same to Cargo.) You can
-  determine your host's target triple with `rustc -vV`. On *nix, the following
+- It's recommended that the `--target` option is always used for `xargo`. This is because it must
+  be provided even when compiling for the host platform due to the way cargo
+  handles compiler plugins (e.g. `serde_derive`) and build scripts (`build.rs`). This also applies to how
+  all of the dependant crates get compiled that use compiler plugins or build scripts.
+  You can determine your host's target triple with `rustc -vV`. On *nix, the following
   rune will extract the triple:
   `rustc -vV | egrep '^host: ' | sed 's/^host: //'`.
+
 
 ## License
 

--- a/README.md
+++ b/README.md
@@ -332,7 +332,6 @@ stage = 2
   rune will extract the triple:
   `rustc -vV | egrep '^host: ' | sed 's/^host: //'`.
 
-
 ## License
 
 Licensed under either of

